nautobot/docs/release-notes/version-1.2.md

```diff
@@ -171,6 +171,7 @@ Just as with the UI, the `slug` can still always be explicitly set if desired.
 - [#1220](https://github.com/nautobot/nautobot/issues/1220) - Fixed an inconsistency in the breadcrumbs seen in various Admin pages.
 - [#1228](https://github.com/nautobot/nautobot/issues/1228) - Fixed a case where a GraphQL query for objects associated by Relationships could potentially throw an exception.
 - [#1229](https://github.com/nautobot/nautobot/pull/1229) - Fixed a template rendering error in the login page.
+- [#1233](https://github.com/nautobot/nautobot/issues/1233) - Prevented a job aborting when an optional ObjectVar is provided with a value of None
```

Comment:
This will need to be moved up into a  new `v1.2.5` section - or you can remove it and one of us will take responsibility for adding it appropriately after the PR is merged.